frontend/src/uxbox/main/data/dashboard.cljs

```diff
@@ -255,7 +277,9 @@
   (ptk/reify ::create-draft-file
     ptk/UpdateEvent
     (update [this state]
-      (update state :files assoc (:id data) data))))
+      (-> state
+          (update :files assoc (:id data) data)
+          (update :recent-files update (:project-id data) conj data)))))
```

Comment:
Maybe `update-in` can be used instead of two nested `update`.

Reply:
You're right. I'll change it